code/modules/surgery/organs/eyes.dm

```diff
@@ -405,3 +405,14 @@
 	eye_icon_state = "snail_eyes"
 	icon_state = "snail_eyeballs"
 
+/obj/item/organ/eyes/fly
+	name = "fly eyes"
+	desc = "These eyes seem to stare back no matter the direction you look at it from."
+
+/obj/item/organ/eyes/fly/Insert(mob/living/carbon/M, special = 0)
+	..()
+	ADD_TRAIT(M, TRAIT_FLASH_SENSITIVE, ORGAN_TRAIT)
+
+/obj/item/organ/eyes/fly/Remove(mob/living/carbon/M, special = 0)
+	REMOVE_TRAIT(M, TRAIT_FLASH_SENSITIVE, ORGAN_TRAIT)
+	..()
```

Comment:
```suggestion
	return ..()
```
Preserve the return call, even if it's `null`, unless you have good reasons not to.